tensorflow/compiler/xla/service/gpu/cudnn_fused_conv_rewriter_test.cc

```diff
@@ -40,30 +41,16 @@ class CudnnFusedConvRewriterTest : public HloTestBase {
   }
 
   void TestMatchWithAllTypes(absl::string_view hlo_string) {
-    string alpha_conv_scalar, alpha_side_input_scalar;
-    string elementwise_type;
-    for (absl::string_view type : {"f16", "f32", "f64", "s8"}) {
-      if (type == "s8") {
-        alpha_conv_scalar = "2";
-        alpha_side_input_scalar = "-3";
-        elementwise_type = "f32";
-      } else {
-        alpha_conv_scalar = "0.999994934";
-        alpha_side_input_scalar = "0.899994934";
-        elementwise_type = string(type);
-      }
-      string hlo_resolved_string = absl::StrReplaceAll(
-          hlo_string, {{"INPUT_TYPE", type},
-                       {"ELEMENTWISE_TYPE", elementwise_type},
-                       {"ALPHA_CONV_SCALAR", alpha_conv_scalar},
-                       {"ALPHA_SIDE_INPUT_SCALAR", alpha_side_input_scalar}});
-      string optimized_hlo_string = GetOptimizedHlo(hlo_resolved_string);
+    for (absl::string_view type : {"f16", "f32", "f64"}) {
```

Comment:
Revert the changes on floating-point tests, as it is very difficult to use them to test int8 convolutions.  We created more int8 specific tests.